exercises/simple-cipher/canonical-data.json

```diff
@@ -0,0 +1,123 @@
+{
+  "exercise": "simple-cipher",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Random key cipher",
+      "property": "random",
+      "cases": [
+        {
+          "description": "It has a key made of letters",
+          "property": "key",
+          "input": "any",
+          "matches_regex": "^[a-z]+$"
```

Comment:
We are in the process of enforcing `lowerCamelCase` keys (see #987), so I would suggest to rename the keys (not only this one) to `lowerCamelCase`.